### ISSUE TITLE
ProgressViewOrientation.VERTICAL

### ISSUE THREAD

**Issue by allison-lekwa** [skydoves/ProgressView#10]
The progress view orientation VERTICAL not working

**Comment by skydoves** [comment]
@allison-lekwa 
Hi, did you set a specific value to the `width` and `height` like below?
```gradle
<com.skydoves.progressview.ProgressView
  android:layout_width="35dp"
  android:layout_height="300dp"
  app:progressView_orientation="vertical"
  
  ...
```

**Comment by allison-lekwa** [comment]
@skydoves 
Hello, yes I did. The horizontal orientation worked perfectly, but the vertical does not show any animation

**Comment by skydoves** [comment]
@allison-lekwa 
Hmm.. In the [demo](https://github.com/skydoves/ProgressView/blob/master/app/src/main/java/com/skydoves/progressviewdemo/CustomActivity.kt) activity, it works well.
It seems to be affected by some other situation.

**Comment by allison-lekwa** [comment]
Even in the demo, the vertical orientation did not work.....

**Comment by skydoves** [comment]
The vertical attribute does below internally.
```kotlin
progressView.rotation = 180f
progressView.highlightView.rotation = 180f
```
Could you make the rotation manually `180f` with the `horizontal` option?

**Comment by allison-lekwa** [comment]
I already did, and this are my observation

progressView.highlightView.rotation = 180f
Does not change anything... Also I do not know whether it has to do with the canvas.path

**Comment by siddesh13** [comment]
This vertical progress view does not work on android 7 Nougat, but android 8 onwards it works
do you have any solution how can we show it on android 7

**Comment by skydoves** [comment]
Hi, @siddesh13.
I just tested using API 17~21 devices but it is hard to represent the issue.
Could you build using this [demo project](https://github.com/skydoves/ProgressView/blob/master/app/src/main/java/com/skydoves/progressviewdemo/CustomActivity.kt) and let me the specific device information?

**Comment by skydoves** [comment]
Fixed in the new version `1.1.0`.
Thank you for the issue :)
